lib/chartmogul/csv/invoice.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ChartMogul
+  module CSV
```

Comment:
documenting this [link](https://help.chartmogul.com/hc/en-us/articles/360013881659-Import-billing-system-data-using-CSV-upload#h_e4e19bd1-38a1-432d-a8f0-9f4b52bee0d7) might be a good idea around here.